pay-id-api-spec/pay-id.v1.yml

```diff
@@ -33,6 +33,43 @@ paths:
           description: Internal error
       operationId: resolvePayID
       description: Resolve a pay id
+  '/{path}/invoice':
+    parameters:
+      - schema:
+          type: string
+        name: path
+        in: path
+        required: true
+    get:
+      summary: get-invoice
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/xrpl-devnet+json:
+              schema:
+                $ref: '#/components/schemas/SignatureWrapper'
+            application/xrpl-mainnet+json:
+              schema:
+                $ref: '#/components/schemas/SignatureWrapper'
+            application/xrpl-testnet+json:
+              schema:
+                $ref: '#/components/schemas/SignatureWrapper'
```

Comment:
nit: Order of `devnet`, `testnet`, `mainnet`. My mind wants them to be sorted somehow.

Reply:
they are sorted alphabetically, by design. 

I could sort them by "seriousness of network" but it's unclear where you'd slot `application/btc-mainnet+json into that.